### DISCUSSION TITLE
`:darktheme` and `:lighttheme` commands that limit theme selections

### DISCUSSION THREAD

**Discussion by phillipspc** [helix-editor/helix#4828] [Ideas]
I like using the `:theme` command and tabbing through the various themes from time to time, especially when some new ones have been added. I _don't_ like going from dark themes to light themes though which can be quite jarring and unpleasant on the eyes. I was watching a recent YT video on helix and this "issue" was brought up there as well: https://youtu.be/i_qsYALaFVo?t=143 (so I guess I'm not the only one?)

My idea would be twofold:
- categorize each them as either a dark or light theme (if that isn't already the case)
- provide additional commands that scope to that subset of themes. ie. I can browse dark themes only by using `:darktheme`

**Comment by braindefender** [comment]
I had an idea to somehow allow the use of regular expressions for theme filtering. That should allow easy theme comparison.
Example: `:theme tokyonight|catppuccin_mocha` will filter out only two themes from list that can be switched back and forth via `Tab`
